spotify-conversion-next/pages/_app.tsx

```diff
@@ -1,12 +1,14 @@
 import { SiteLayout } from "@/components";
 import "@/styles/globals.css";
-import { ChakraProvider } from "@chakra-ui/react";
+import { Box, ChakraProvider, Container } from "@chakra-ui/react";
 import type { AppProps } from "next/app";
 
 export default function App({ Component, pageProps }: AppProps) {
   return (
     <ChakraProvider>
-      <Component {...pageProps} />
+      <Container maxW="1200px" justifyContent="center">
```

Comment:
This is why every page looks skinnier. I maxed out the width of our pages at 1200px... that's pretty standard.